### ISSUE TITLE
Usage of the translation library

### ISSUE THREAD

**Issue by marado** [shawncplus/ranviermud#51]
The translation library is being used in a subverted way. The idea behind the library is that you have translations like:

```
Welcome!:
  es: "Bienvenido!"
  pt: "Bem vindo!"
```

And then you set the defaultLanguage as `en`, getting:

```
 console.log(myLocalize.translate("Welcome!")); // Welcome!
 myLocalize.setLocale("es");
 console.log(myLocalize.translate("Welcome!")); // Bienvenido!
```

ranviermud subverts this, setting the default language as "zz", and forcing every string to have an "en" translation. Doing this makes us loose a powerful tool: the ability of having things untranslated. If we were using the library as meant, this would happen:

```
 myLocalize.setLocale("de");
 console.log(myLocalize.translate("Welcome!")); // Welcome!
```

Since there's no german translation to "Welcome!", it would fallback to the default language. Right now, that can't happen (and that explains why we have an unusable Spanish translation). This problem will only grow: what happens when we have full support for a language, but then we want to accept patches that introduce strings for which we still don't have translation? 

All the above made me want to start working on fixing this, using the translation library as it is supposed to. There's a bit of a problem with that, tho: we started depending on the mechanism as it is. Check `rooms.yml` for instance: we _depend_ on the sentence on the original language being "description" and not the actual sentence, for each room. 

So, in summary:
- we should start using the translation library as it was designed to: we'll reap benefits, and as the project grows the current state keeps getting more unmanageable. But, in order to do so,
- we should re-write the automatism mechanisms we have already in place, that are assuming the "wrong way of doing things" as being the way things will be. That means not a small undertaking, rewriting the Building mechanism.
